example/lib/main.dart

```diff
@@ -111,6 +111,7 @@ class _MyAppState extends State<MyApp> {
           ..setRecordAppStartTime(true) // Enable APM features, which includes the recording of app start time.
           ..setStarRatingTextMessage('Message for start rating dialog')
           ..setLoggingEnabled(true) // Enable countly internal debugging logs
+          ..enrollABOnRCDownload()
```

Comment:
let's comment this out. Lets add a coment that this should be used in specific circumstances